### PR TITLE
Picocli

### PR DESCRIPTION
**Pull request by yschimke**
Better mix of features and better for testing graal nativeImage

### CODE REVIEW

**Comment by yschimke** [comment]
image

**Comment by yschimke** [comment]
`./okcurl --completionScript "a"` will generate a bash tab completion script

**Comment by yschimke** [comment]
@swankjesse Thoughts on this?  It mainly helps as a testbed for confirming graal compatibility for OkHttp.  That is useful for me, I ended up porting a CLI client to python requests and I'd prefer to stay within a single codebase.

File: okcurl/okcurl [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -e
+
+../gradlew -q --console plain installDist
```
Comment (swankjesse) [inline okcurl/okcurl]: What does this do?

File: build.gradle [hunk 2]
```diff
@@ -18,11 +17,12 @@ buildscript {
       'kotlin': '1.3.31',
       'moshi': '1.8.0',
       'okio': '2.2.2',
-      'ktlint': '0.31.0'
+      'ktlint': '0.31.0',
+      'picocli': '4.0.0-beta-1b'
```
Comment (swankjesse) [inline build.gradle]: Should we wait for a proper release?

Comment (yschimke) [reply build.gradle]: Sounds good, I'll put this on a branch or similar until then.  Test with that.

File: okcurl/okcurl (hunk 1 above)
Comment (yschimke) [reply okcurl/okcurl]: silently builds (with gradle caching etc), so that you can run "./okcurl https://..." from a git checkout and it *just* works

**Pull request merged**